docs/docs/reaching-out-to-user.mdx

```diff
@@ -0,0 +1,459 @@
+getting the conversation ID back from the assistant:
+
+```yaml
+User: /ask_id
+  Bot: The ID of this conversation is user1234.
+      Trigger an intent with
+      curl -H "Content-Type: application/json" -X POST -d
+      '{"name": "EXTERNAL_dry_plant", "entities": {"plant": "Orchid"}}'
+      "http://localhost:5005/conversations/user1234/trigger_intent?output_channel=latest"
+```
+
+By running the returned POST request,
+you should see the bot respond in your channel:
+
+```yaml
+  Bot: Your Orchid needs some water!
+```
+
+
+## Reminders
+
+You can have your assistant reach out to the user after a set amount of time by using **reminders**.
+The examples below are from the [reminderbot example bot](https://github.com/RasaHQ/rasa/blob/master/examples/reminderbot).
+You can clone it and follow the instructions in `README` to try out the full version.
+
+### Scheduling Reminders
+
+To schedule a reminder, you need to define a custom action that returns
+the `ReminderScheduled` event. For example, the following custom action
+schedules a reminder for five seconds from now:
+
+```python
+class ActionSetReminder(Action):
+    """Schedules a reminder, supplied with the last message's entities."""
+
+    def name(self) -> Text:
+        return "action_set_reminder"
+
+    async def run(
+        self,
+        dispatcher: CollectingDispatcher,
+        tracker: Tracker,
+        domain: Dict[Text, Any],
+    ) -> List[Dict[Text, Any]]:
+
+        dispatcher.utter_message("I will remind you in 5 seconds.")
+
+        date = datetime.datetime.now() + datetime.timedelta(seconds=5)
+        entities = tracker.latest_message.get("entities")
+
+        reminder = ReminderScheduled(
+            "EXTERNAL_reminder",
+            trigger_date_time=date,
+            entities=entities,
+            name="my_reminder",
+            kill_on_user_message=False,
+        )
+
+        return [reminder]
+```
+Note that this action requires the `datetime` and `rasa_sdk.events` packages.
+
+The first argument for the `ReminderScheduled` event is the reminder's name, in this case, `EXTERNAL_reminder`.
+The reminder name will be used later as an intent to trigger a reaction to the reminder.
+As with external events, it's recommended to name the reminder name with the
+`EXTERNAL_` prefix to make it easier to see what's going on in your training data.
+
+You can see that the last messages' `entities` are also passed to the reminder.
+This allows the action that reacts to the reminder to make use of the entities
+from the user's scheduling message.
+
+For example, if you want your assistant to remind you to call a friend, you could
+send it a message like "Remind me to call Paul". If `Paul` is extracted as a `PERSON`
+entity, the action reacting to the reminder can use it to say "Remember to call Paul!"
+
+### Reacting to Reminders
+
+As with external events, the bot reaches out to the user after receiving a
```

Comment:
should we remove all the "as with external events" from this section? I feel like it doesn't add anything, and it's not a given that people will have read the previous section

Reply:
Ja there is a logical link between the because of the trigger endpoint, but I don't think that matters to the user reading it